dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05370ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,355 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.datagen.TemplateDataGen;
+import com.dotcms.datagen.ThemeDataGen;
+import com.dotcms.rendering.velocity.viewtools.DotTemplateTool;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.templates.model.Template;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class Task05370ChangeContainerPathToAbsoluteTest {
+    final String body =
+        "<html>" +
+            "<head>" +
+                "#dotParse('%1$s/application/themes/landing-page/html_head.vtl')" +
+                "<link rel=\"stylesheet\" type=\"text/css\" href=\"/html/css/template/reset-fonts-grids.css\" />" +
+            "</head>" +
+            "<body>" +
+                "<div id=\"resp-template\" name=\"globalContainer\">" +
+                    "<div id=\"hd-template\">" +
+                    "#dotParse('%1$s/application/themes/landing-page/header.vtl')" +
+                "</div>" +
+                "<div id=\"bd-template\">" +
+                    "<div id=\"yui-main-template\">" +
+                        "<div class=\"yui-b-template\" id=\"splitBody0\">" +
+                            "#parseContainer('%1$s/application/containers/default/','1')" +
+                        "</div>" +
+                    "</div>" +
+                "</div>" +
+            "</body>" +
+        "</html>";
+
+    final String jsonDrawBody = "{" +
+            "\"title\": \"layout_test\"," +
+                "\"body\": {" +
+                    "\"rows\": [" +
+                        "{" +
+                            "\"columns\": [" +
+                                "{" +
+                                    "\"containers\": [" +
+                                        "{" +
+                                            "\"identifier\": \"%s/application/containers/default/\"," +
+                                            "\"uuid\": \"1\"" +
+                                        "}" +
+                                    "]," +
+                                    "\"widthPercent\": 83," +
+                                    "\"leftOffset\": 1," +
+                                    "\"width\": 10," +
+                                    "\"left\": 0" +
+                                "}" +
+                            "]" +
+                        "}" +
+                    "]" +
+            "}" +
+        "}";
+
+    final String legacyHTMLLayout = "" +
```

Comment:
Codacy found an issue: [Do not add empty strings](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5990100)